helper/colorlogger.py

```diff
@@ -14,7 +18,7 @@ def create_logger(name, color='reset', log_level=logging.INFO, log_colors={
     handler.setFormatter(colorlog.ColoredFormatter('%(log_color)s%(asctime)s %(' + color + ')s[%(module)10s] %(log_color)s[%(levelname)5s] %(' + color + ')s%(message)s',
                                                    log_colors=log_colors))
     log = colorlog.getLogger(name)
-    log.propagate = False
+    log.propagate = True
```

Comment:
Well, if you're careful about it. If you set another formatter, then you can get duplicate logging which can be hard for you and (others!) to track down in the code.